specs/core/0_beacon-chain.md

```diff
@@ -1602,7 +1600,7 @@ For every `shard_committee_at_slot` in `state.shard_committees_at_slots` and for
 
 First, we define some additional helpers:
 
-* Let `base_reward_quotient = BASE_REWARD_QUOTIENT * integer_squareroot(total_balance // GWEI_PER_ETH)`.
+* Let `base_reward_quotient = BASE_REWARD_QUOTIENT * integer_squareroot(total_balance)`.
```

Comment:
the `BASE_REWARD_QUOTIENT` is actually selected to expect the int_sqrt to be of total_balance in eth rather than GWEI. We need to modify `BASE_REWARD_QUOTIENT` if we choose to not normalize `total_balance` to ETH

Reply:
Oh yeah. It corresponds to the 10 million participating ETH in every epoch. Should we just keep `GWEI_PER_ETH` for `integer_squareroot`?

Reply:
I'm tempted to modify `BASE_REWARD_QUOTIENT` for Gwei accordingly.

Reply:
yeah, I think that's the way to go.

A  comparable  interest rate is when `BASE_REWARD_QUOTIENT` is set to `1 / 32`.
If we do this, we would need to reward this calculation to be `integer_squareroot(total_balance) // BASE_REWARD_QUOTIENT`.